src/navs/index.js

```diff
@@ -1891,9 +1891,43 @@ export const docsMenu = {
                 },
                 {
                     name: 'Trends',
-                    url: '/docs/product-analytics/trends',
+                    url: '/docs/product-analytics/trends/overview',
```

Comment:
If you make `/docs/product-analytics/trends/overview` -> `index.mdx`, this can just be `/docs/product-analytics/trends/`